src/components/Form/Receipt/contents/NyttigeLenker.tsx

```diff
@@ -0,0 +1,47 @@
+import { ReactElement } from 'react'
+import { BodyLong, Box, Heading } from '@navikt/ds-react'
+
+import { TrackedExternalLink } from '@/components/Link/TrackedExternalLink'
+
+const NyttigeLenker = (): ReactElement => (
```

Comment:
Hm, burde det her være en liste? Enten designsystem-liste, eller en tailwind liste? (De har litt ulikt design).

Det er vel noe med html semantikk som kanskje taler til fordel for liste med en slik "oppramsing"

Reply:
Ja, kanskje. Det er jo en oppramsing ja. Prøver...